### ISSUE TITLE
Apply custom filter button on Dashboard is disabled after one selection and doesn't get enabled when user makes an other selection.

### ISSUE THREAD

**Issue by raghupola-vg** [hzi-braunschweig/SORMAS-Project#3465]
### Bug Description
When a custom date is selected on Dashboard and clicked on Apply Custom Filter, it is disabled and even when a user changes the date again this cannot be altered unless user clicks on reset filters.

### Steps to Reproduce
1. Login to sormas as any of the users who has access to dashboard.
2.  Click on epi week selection and Select Custom.
3. Pick a Custom date and click on Apply custom Filter.
4. Now change the date/week and see if the Apply Custom Filter button is enabled or not.

### Expected Behavior
Apply Custom Filter button must appear active every time a change is made and user should have the option to apply these changes.

at the moment user can change the selections only after resetting all filters and then making a re-selection.

### Screenshots
![Apply custom filter_dashboard_disabled](https://user-images.githubusercontent.com/62763628/98914034-77fc1380-24c8-11eb-9358-b4b0734c80f7.PNG)

### System Details
* Device:
* SORMAS version: 1.52.0-snapshot
* Android version/Browser: Chrome
* Server URL: https://test.sormas.netzlink.com/
* User Role: SurvSup

**Comment by bernardsilenou** [comment]
In addition:
apply custom filter does not filter the dashboard when clicked on. You need to later click on the apply filter icon a gain.
I think the apply custom filter icon is redundant and confusing. Can we keep only "apply filter" icon?
When custom icon is activated, then the "apply filter"  method will use the variables parsed by the custom filter . if not we use the default variables.
Each time the user click on "apply filter", system should reset the filter and use the new values supplied by the user.

**Comment by AndyBakcsy-she** [comment]
Verified issue on SORMAS Version: 1.53.0-SNAPSHOT

**Comment by vidi42** [comment]
@bernardsilenou  please open a new issue for your comment
